src/main/java/uk/gov/hmcts/reform/idam/web/strategic/StepUpAuthenticationZuulFilter.java

```diff
@@ -31,10 +31,12 @@
 
     private final SPIService spiService;
     private final String idamSessionCookieName;
+    private final boolean useStepUpAuthentication;
```

Comment:
Can we replace this with:
{code}
@ConditionalOnProperty("feature.step-up-authentication")
{code}

Reply:
Done.